recipe/meta.yaml

```diff
@@ -1,34 +1,32 @@
 {% set name = "conda-package-handling" %}
-{% set version = "1.9.0" %}
+{% set version = "2.0.2" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   url: https://github.com/conda/conda-package-handling/archive/{{ version }}.tar.gz
-  sha256: 81e0771b8b10ab5c20678a5da714e36d97e321acfc3e5b304ec2e650ce84734b
+  sha256: ec0a2c51f442d98b935ee2288beb1ec6c7e25020b5961f8d09f35738b276b532
 
 build:
-  number: 1
+  number: 0
   script: {{ PYTHON }} -m pip install . --no-deps -vv
   entry_points:
     - cph = conda_package_handling.cli:main
 
 requirements:
   build:
-    - {{ compiler('c') }}
+    - python                                 # [build_platform != target_platform]
```

Comment:
this stuff needs to be removed.  we do not have build_platform being defined ...  no cross compilation ...

Reply:
Will we ever be able to keep our recipes more similar to conda-forge, won't this conditional always be false anyway?